pytorch_lightning/core/lightning.py

```diff
@@ -1756,14 +1759,16 @@ def to_onnx(
         self,
         file_path: Union[str, Path],
         input_sample: Optional[Any] = None,
+        model_check: Callable = None,
         **kwargs,
-    ):
+    ) -> None:
         """
         Saves the model in ONNX format
 
         Args:
             file_path: The path of the file the onnx model should be saved to.
             input_sample: An input for tracing. Default: None (Use self.example_input_array)
+            model_check: custom function that conducts onnx model checks.
```

Comment:
so we want to make the name reflect it is a function? append _fn or _fx?

Reply:
Sounds good to me, whichever is the more common convention in the rest of the Lightning code base would be fine